spec/models/racer_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+# t.string "first_name"
+# t.string "last_name"
+# t.string "gender"
+# t.string "email"
+# t.string "city"
+# t.string "state"
+# t.date "birth_date"
```

Comment:
Dave Mitchell taught me to include the schema as comments in the model spec, just for reference. They perform no function other than quick reference.